src/Honeybee_orientHBGlz.py

```diff
@@ -0,0 +1,71 @@
+#
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
+# 
+# This file is part of Honeybee.
+# 
+# Copyright (c) 2013-2016, Mostapha Sadeghipour Roudsari <Sadeghipour@gmail.com> 
+# Honeybee is free software; you can redistribute it and/or modify 
+# it under the terms of the GNU General Public License as published 
+# by the Free Software Foundation; either version 3 of the License, 
+# or (at your option) any later version. 
+# 
+# Honeybee is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+
+"""
+Use this component to set the orientation(s) of the glazing.
+
+-
+Provided by Honeybee 0.0.59
+
+    Args:
+        _orientation: The orientation/s that you'd like to add glazing to.
+        _ratio: The window to wall ratio.
+    Returns:
+        _glzratio: A list of glazing orientations that can be plugged into (glazingCreator).
+"""
+
+import rhinoscriptsyntax as rs
```

Comment:
You are not using any of this libraries. I would just remove them from the script.